WordPress/WordPress.xcodeproj/project.pbxproj

```diff
@@ -18780,6 +22459,279 @@
 			};
 			name = "Release-Alpha";
 		};
+		FABB264E2602FC2C00C8785C /* Debug */ = {
+			isa = XCBuildConfiguration;
+			baseConfigurationReference = 152F25D5C232985E30F56CAC /* Pods-Jetpack.debug.xcconfig */;
+			buildSettings = {
+				ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES = "$(inherited)";
+				APPLICATION_EXTENSION_API_ONLY = NO;
+				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
+				CLANG_ENABLE_MODULES = "$(inherited)";
+				CLANG_ENABLE_OBJC_ARC = YES;
+				CLANG_WARN__ARC_BRIDGE_CAST_NONARC = NO;
+				CODE_SIGN_ENTITLEMENTS = WordPress.entitlements;
+				CODE_SIGN_IDENTITY = "iPhone Developer";
+				COPY_PHASE_STRIP = NO;
+				DEFINES_MODULE = YES;
+				DEVELOPMENT_TEAM = PZYM8XX95Q;
+				ENABLE_BITCODE = NO;
+				FRAMEWORK_SEARCH_PATHS = "$(inherited)";
+				GCC_OPTIMIZATION_LEVEL = 0;
+				GCC_PREFIX_HEADER = WordPress_Prefix.pch;
+				GCC_PREPROCESSOR_DEFINITIONS = (
+					"$(inherited)",
+					"COCOAPODS=1",
+					"NSLOGGER_BUILD_USERNAME=\"${USER}\"",
+					"WPCOM_SCHEME=\\@\\\"${WPCOM_SCHEME}\\\"",
+					"JETPACK=1",
```

Comment:
If you're specifying this here, you might consider also specifying it under "other swift flags" in order to have it available in Swift as well (without having to do any bridging for this special case).

Reply:
Fixed! 7038bc1